test-lib.sh

```diff
@@ -865,20 +865,25 @@ ct_test_app_dockerfile() {
   echo "Using this Dockerfile:"
   cat Dockerfile
 
-  # If app_url contains @, the string after @ is considered
-  # as a name of a branch to clone instead of the main/master branch
-  IFS='@' read -ra git_url_parts <<< "${app_url}"
+  if  [[ $app_url == *".git"* ]]; then
```

Comment:
Are we ssure all git URI end with ".git"? I'm a bit sceptical. Why not switch it around, first test `[ -d "$app_url" ]`, and treat everything else as git URI as before? A git URI will never be an existing local directory.

Reply:
You are right, implemented.